python/pyarrow/feather.py

```diff
@@ -29,74 +29,6 @@ def _check_pandas_version():
         raise ImportError("feather requires pandas >= 0.17.0")
 
 
-class FeatherReader(ext.FeatherReader):
```

Comment:
Any idea if there would be people using those classes?

(I have no problem with removing them directly, though, but we could also deprecate first)

Reply:
I don't think anyone should be using them. Some of the features that they used have been removed